### PR TITLE
Datarequest

### PR DESCRIPTION
**Pull request by aucuparia**

### CODE REVIEW

**Review by lwesterhof — APPROVED** [review]
LGTM

**Review by lwesterhof — APPROVED** [review]

File: datarequest/schemas/youth-0/datarequest/schema.json [hunk 1]
```diff
@@ -29,9 +29,13 @@
           "items": {
             "type": "object",
             "properties": {
-              "name": {
+              "first_name": {
```
Comment (lwesterhof) [inline datarequest/schemas/youth-0/datarequest/schema.json]: In default-1 we use first name as label, but the field is named given_name. To be consistent with DataCite. And Given name and Family Name may also be a nicer combination.

**Review by lwesterhof — CHANGES_REQUESTED** [review]

File: datarequest/schemas/youth-0/datarequest/schema.json (hunk 1 above)
Comment (aucuparia) [reply datarequest/schemas/youth-0/datarequest/schema.json]: Agreed. Fixed.

**Review by lwesterhof — APPROVED** [review]

**Review by lwesterhof — APPROVED** [review]

**Review by lwesterhof — APPROVED** [review]

**Review by lwesterhof — APPROVED** [review]

**Review by lwesterhof — APPROVED** [review]

**Review by lwesterhof — APPROVED** [review]

**Review by lwesterhof — CHANGES_REQUESTED** [review]
Several new API have been introduced, is it possible to add some API tests for these?

Would also be interesting to have one complete happy flow of the module as UI test.

**Comment by aucuparia** [comment]
> Several new API have been introduced, is it possible to add some API tests for these?

Done.

**Review by lwesterhof — APPROVED** [review]

**Review by lwesterhof — APPROVED** [review]

**Review by lwesterhof — APPROVED** [review]

**Pull request merged**